src/layouts/sidebar-sidecar/types.ts

```diff
@@ -12,6 +12,8 @@ interface BaseProps {
   children: React.ReactNode
   githubFileUrl?: string
   openConsentManager?: () => void
+  // TODO: wip
+  sidebarPropsLevels?: SidebarProps[]
```

Comment:
I think we might want to change the type of `sidebarProps` since it will become unneeded anyways, but for the moment I created a separate prop and made the layout backwards compatible with `sidebarProps`.

Reply:
Removed, no longer relevant! ✨